Woof/Woof/Model/LocalStorage/LocalStorageKey.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+/// Possible keys used for storing data in persistent data storage.
```

Comment:
So I can't use any other keys except the `LocalStorageKey`?

Reply:
Removed as unnecessary

Reply:
We can store any key in our KeyValueStorage. 
The was removed as unnecessary.